Sources/PostgreSQL/Connection.swift

```diff
@@ -152,6 +152,15 @@ public final class Connection: ConnInfoInitializable {
         public let channel: String
         public let payload: String?
         
+        /// initializer usable without knowledge of CPostgreSQL
+        /// required to allow unit testing of classes using Notifications
+        public init(pid: Int, channel: String, payload: String?) {
```

Comment:
You do not need to make this `init` public. If you do `@testable import`, it should just work. This struct should not be initializable outside of this library.

Reply:
I'll remove that change